src/freenet/clients/fcp/ClientGet.java

```diff
@@ -925,14 +924,17 @@ public boolean filterData() {
 	synchronized RequestStatus getStatus() {
 		boolean totalFinalized = false;
 		int total = 0, min = 0, fetched = 0, fatal = 0, failed = 0;
+		Date latestSuccess = CurrentTimeUTC.get(), latestFailure = null;
```

Comment:
> Why does latestSuccess start at the current time? Is my assumption correct that getStatus() may be called (and indicate progress) without progressPending being set?

Commit 2cea4dd24242562d1e3b54951a8b0ac462f26dee fixes this and similar occurrences to point the reader to the JavaDoc which explains the reason for the default value.